<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-BGS5S7GJ" ignored="0" name="aaa" tests="2" failures="1" timestamp="13 六月 2020 07:25:05 GMT" time="40.358" errors="0">
  <testcase name="testKeword" time="12.021" classname="com.crazy.auto.keyword.KeywordTestsMobile"/>
  <testcase name="testKeword" time="13.652" classname="com.crazy.auto.keyword.KeywordTestsMobile">
    <failure type="java.lang.AssertionError" message="退出失败">
      <![CDATA[java.lang.AssertionError: 退出失败
at com.crazy.auto.keyword.KeywordTestsMobile.testKeword(KeywordTestsMobile.java:54)
Caused by: java.lang.NullPointerException
at com.crazy.auto.element.ElementOperate.click(ElementOperate.java:108)
at com.crazy.auto.keyword.KeyWordsAction.click(KeyWordsAction.java:160)
at com.crazy.auto.keyword.KeywordTestsMobile.excuteKeyword(KeywordTestsMobile.java:70)
at com.crazy.auto.keyword.KeywordTestsMobile.testKeword(KeywordTestsMobile.java:50)
... 25 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testKeword -->
</testsuite> <!-- aaa -->
